config/checkstyle/checkstyle.xml

```diff
@@ -385,10 +385,10 @@
     JAVADOC CHECKS
     -->
 
-    <!-- Checks that all block-tags are ordered correctly. -->
+    <!-- Checks that all block-majors are ordered correctly. -->
     <module name="AtclauseOrder"/>
 
-    <!-- Checks that Javadoc block tags appear only at the beginning of the line. -->
+    <!-- Checks that Javadoc block majors appear only at the beginning of the line. -->
```

Comment:
These changes should be reverted too back into tags

Reply:
Ok thank you it has been resolved.